src/damavand/base/controllers/spark.py

```diff
@@ -158,13 +150,13 @@ def application_with_id(self, app_id: str) -> Sparkle:
         raise ValueError(f"Application with ID {app_id} not found.")
 
     @runtime
-    def run(self, app_id: str) -> None:
+    def run_application(self, id_: str) -> None:
```

Comment:
app_id naming could be less confusing I think. id_ might get confused with the controller id which is called in the same way